partition/README.md

```diff
@@ -0,0 +1,33 @@
+# partition
+
+Contains roles for deploying the metal-partition.
+
+## Requirements
+
+- [ansible-common](https://github.com/metal-stack/ansible-common)
+
+## Variables
+
+The `parition-defaults` folder contains defaults that are used by multiple roles in the control-plane directory.
+
+You can look up all the default values [here](partition-defaults/main.yaml).
+
+| Name                                | Mandatory | Description                                                                                                              |
+| ----------------------------------- | --------- | ------------------------------------------------------------------------------------------------------------------------ |
+| router_nameservers                  | yes       | A list of nameservers                                                                                                    |
+
+
+## Roles
+
+| Role Name                                                                | Description                                                                                                                     |
+| ------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------- |
+| [dhcp](roles/dhcp)                                                       | Deploys a dhcp server                                                                                                           |
+| [docker-on-cumulus](roles/docker-on-cumulus)                             | Deploys docker on cumulus                                                                                                       |
+| [ipmi-catcher](roles/ipmi-catcher)                                       | Deploys ipmi-catcher to crawl ipmi ip addresses                                                                                 |
+| [leaf](roles/leaf)                                                       | Deploys network config for cumulus switches                                                                                     |
+| [pixicore](roles/pixicore)                                               | Deploys pixicore                                                                                                                |
```

Comment:
The project is called "pixiecore"
https://github.com/danderson/netboot/tree/master/pixiecore